package.json

```diff
@@ -91,6 +95,8 @@
     "@types/sketchapp": "^1.0.0",
     "codelyzer": "~4.5.0",
     "conventional-changelog-cli": "^2.0.12",
+    "cypress": "^3.1.5",
```

Comment:
We should be including cypress as a dependency because of this issue: https://github.com/xlayers/xlayers/issues/275

See related PR: https://github.com/xlayers/xlayers/commit/b9a2676ab51301b7eb510167face85f3682e22de